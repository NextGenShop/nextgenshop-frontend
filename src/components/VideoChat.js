import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Paper from "@material-ui/core/Paper";
import AccountCircleIcon from "@material-ui/icons/AccountCircle";

const useStyles = makeStyles((theme) => ({
  paper: {
    display: "flex",
    color: theme.palette.text.secondary,
    background: "#eeeeee",
    height: "75vh",
    alignItems: "center",
    justifyContent: "center",
  },
}));

export default function VideoChat({ items }) {
  const classes = useStyles();
  return (
    <Paper className={classes.paper}>
      <AccountCircleIcon />
    </Paper>
  );
}
