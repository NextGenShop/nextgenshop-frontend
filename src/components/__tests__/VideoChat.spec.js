import React from 'react';
import { shallow } from 'enzyme';
import VideoChat from '../VideoChat';

describe('VideoChat', () => {
  test('should correctly render', () => {
    const component = shallow(<VideoChat />);
    expect(component).toMatchSnapshot();
  });
});
