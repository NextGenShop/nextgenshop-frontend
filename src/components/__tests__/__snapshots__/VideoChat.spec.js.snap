// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoChat should correctly render 1`] = `
<WithStyles(ForwardRef(Paper))
  className="makeStyles-paper-1"
>
  <Memo([object Object]) />
</WithStyles(ForwardRef(Paper))>
`;
